htdocs/fichinter/card.php

```diff
@@ -1386,7 +1386,7 @@
 
 					$line->fetch_optionals();
 
-					print $line->showOptionals($extrafields, 'view', array('colspan'=>5));
+					print $line->showOptionals($extrafields, 'view', array('style' => 'class="nowrap"', 'colspan' => 5));
```

Comment:
Can you provide a screenshot of what you have with bugged version ?

Reply:
![what_the_fuck_fichinter](https://user-images.githubusercontent.com/881589/146677272-d863d277-e456-40b8-bc15-a561746bdf90.jpg)

Reply:
Maybe releaxtion #19600?

Reply:
But for me version 15 ok
![7BA082AE-0306-49A9-8E77-6B806D79A704](https://user-images.githubusercontent.com/29521447/147307770-42590862-5469-4c77-8b9b-9181889d4950.png)

Reply:
But your document is not a ficheinter